jre_emul/Makefile

```diff
@@ -206,8 +206,8 @@ dist_includes: $(BUILD_DIR)/.jre_dist_includes
 $(BUILD_DIR)/.jre_dist_includes: $(PUBLIC_TRANSLATED_HEADERS:%=$(GEN_OBJC_DIR)/%) \
     $(PUBLIC_EMULATION_HEADERS:%=$(EMULATION_CLASS_DIR)/%) | $(DIST_INCLUDE_DIR)
 	@echo copying jre_emul header files
-	@(cd $(EMULATION_CLASS_DIR) && tar cf - \
-	    $(PUBLIC_EMULATION_HEADERS:$(EMULATION_CLASS_DIR)/%=%)) \
+	@cd $(EMULATION_CLASS_DIR) && tar cf - \
```

Comment:
Ok, I don't think that fix actually fixed anything (same problem)
copying jre_emul header files
tar: Unrecognized archive format
tar: Error exit delayed from previous errors.
make[1]: *** [/Users/mhawkins/src/j2objc/jre_emul/build_result/.jre_dist_includes] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [jre_emul_dist] Error 2

I'm guessing the files were copied there when I ran the command manually... which allowed the build to finish.

If I ONLY change that line, the files aren't copied, but the "copy headers" phase completes.  The problem later comes when 'IOSClass.h' isn't found... because it wasn't actually copied to dist/include